### DISCUSSION TITLE
Is there any way to plot bigger plots?

### DISCUSSION THREAD

**Discussion by a9lara** [fonsp/Pluto.jl#1158] [Questions about using Pluto] [answered]
Hello. 

Maybe this question is regarding more about Plots.jl than Pluto.jl.

I am using Plots.jl along with plotly on Pluto.jl. I understand the format of Pluto's page, but when it comes to plots, it becomes not very useful because it displays the plots in small size and the numbers of the yticks are covered by the ylabel. Also, I cant make latex work on the labels. The interaction with the plot s is amazing, but not the visual part.

Well, as you can see I am having a lot of issues with Plots.jl. I am wondering if it is only Plots or it has something to do with Pluto.

I am also wondering if there is any way to prompt the plot outside Pluto.

What advice can you give me? What is the best way to plot clean and complete on Pluto?

Regards

Armando

**Comment by dorn-gerhard** [comment, marked as answer]
Hey Armando,
I also use a lot of Plots in Pluto Notbooks (using the backend gr)
I think the problem does not relate to Pluto but to the Plots package.
It took me a while to figure out how to make nice plots with Pluto / Julia and here is some example which might help you regarding plots:

Let's say, you want to make a simple line plot of the function sin(x) with proper Latex labelling.

**LaTeX:** The first clever trick is to use the latex font Computer modern - then your text looks like LaTeX. 
The package LaTeXStrings is very useful if you want to use formulas (not all Latexformulas work acually -> e.g. \boldsymbol{x} won't work (as far i know), but just ask in the forum of gr - they can solve issues with latex in their backend

**Plot size** You can save plots and then open them, if this is of help for you - the option to have a plot opened in a new tab would be nice or to have adjustable cell sizes -> I think there are plans that go in that direction

**cropped margins, font size, etc** Below is a test file with some useful commands, where you can adjust image size, the borders, font sizes, background transparency, and so on. More options are to discovered in the attributes section of JuliaPlots: http://docs.juliaplots.org/latest/generated/attributes_plot/

Best, 
Gerhard

```
begin
	
using Plots
using Plots.PlotMeasures
using LaTeXStrings

t = 0:0.1:2*pi

plot(t, sin.(t),
	title = "A sine curve",
	# labels
	xlabel = latexstring("a_{\\sphericalangle}"), 
	ylabel = latexstring("p\\,(a_{\\sphericalangle})"),
	label = latexstring("\\int_0^\\infty \\sin(x) dx = ?"),
	# series properties
	linewidth = 3,
	# size of image, margins, font size, ...
	size = (600,400),  
	labelfontsize = 30,
	legendfontsize = 12,
	tickfontsize = 15,
	bottom_margin =25mm,
	left_margin = 5mm,
	right_margin = 10mm,
	titlefontsize = 20,	
	background_color = :transparent,  
	# this is really cool, transparent background :)
	foreground_color = :black,
	# makes it look like LaTeX
	fontfamily="Computer Modern"
)



#If you want to save the image use the following command (for one backslash (windows) you can enter two slashes), for convenience 
you could also use a print button here :)
	
#savefig("C://nice_pluto_plots//sine_curve")
	
end
```
![test_plot](https://user-images.githubusercontent.com/67096719/118144664-fea89380-b40c-11eb-8760-e52baa17d65b.png)

**Reply by a9lara** [reply]
Wow! Thank you very much! That is amazing!
